Source/CoverageConfiguration.pas

```diff
@@ -644,12 +642,12 @@ procedure TCoverageConfiguration.ParseSwitch(var AParameter: Integer);
   begin
     // do nothing, because its already parsed
   end
+  else if SwitchItem = I_CoverageConfiguration.cPARAMETER_DGROUPPROJ then
+    ParseDgroupProjSwitch(AParameter)
   else if SwitchItem = I_CoverageConfiguration.cPARAMETER_DPROJ then
     ParseDprojSwitch(AParameter)
   else if SwitchItem = I_CoverageConfiguration.cPARAMETER_EXCLUDE_SOURCE_MASK then
-    ParseSourceMaskSwitch(AParameter, {out} FExcludeSourceMaskLst)
-  else if SwitchItem = I_CoverageConfiguration.cPARAMETER_INCLUDE_SOURCE_MASK then
-    ParseSourceMaskSwitch(AParameter, {out} FIncludeSourceMaskLst)
+    ParseExcludeSourceMaskSwitch(AParameter)

```

Comment:
You have removed `cPARAMETER_INCLUDE_SOURCE_MASK`, please fix your code in such a way that is stays backwards compatible.

Reply:
Fixed in last commit